src/demo-app/autocomplete/autocomplete-demo.html

```diff
@@ -66,19 +66,6 @@
   </md-card>
 </div>
 
-<md-autocomplete #reactiveAuto="mdAutocomplete" [displayWith]="displayFn">
```

Comment:
Why were these removed from the demo?

Reply:
They are duplicates of the same elements above.

It look slike we had 5 `<md-autocomplete>`s for 3 `[mdAutocompleteTrigger]`s